frontend/src/pages/PageTestbed.vue

```diff
@@ -10,16 +10,28 @@
 
   <!-- phenogrid -->
   <AppSection>
-    <AppHeading>Phenogrid</AppHeading>
+    <AppHeading>Phenogrid - Search mode</AppHeading>
     <iframe
+      name="pheno-search"
       title="Phenogrid"
       frameBorder="0"
-      src="phenogrid?source=HP:0000939,HP:0000444,HP:0000546,HP:0000135&target_set=Human+Diseases"
+      src="/phenogrid-search?subjects=HP:0000939,HP:0000444,HP:0000546,HP:0000135&object-group=Human+Diseases"
     ></iframe>
 
     <AppButton text="Send Message" @click="sendMessage" />
   </AppSection>
 
+  <!-- phenogrid -->
+  <AppSection>
+    <AppHeading>Phenogrid - Multicompare mode</AppHeading>
+    <iframe
+      name="pheno-multi"
+      title="MultiCompare Phenogrid"
+      frameBorder="0"
+      src="/phenogrid-multi-compare?subjects=HP:0002616,HP:0001763,HP:0004944,HP:0010749,HP:0001533,HP:0002020,HP:0012450&object-sets=HP:0002616,HP:0001763,HP:0000767,HP:0000023,HP:0002108,HP:0000490,HP:0000545,HP:0100785,HP:0000268&object-sets=HP:0002616,HP:0001763,HP:0004944,HP:0010749,HP:0001533,HP:0002020,HP:0012450,HP:0003394,HP:0003771,HP:0012378,HP:0001278,HP:0002827,HP:0002829,HP:0002999,HP:0003010&object-sets=HP:0002616,HP:0001763,HP:0000767,HP:0000023,HP:0002108,HP:0000490,HP:0000545,HP:0100785,HP:0000268,HP:0001634,HP:0001653,HP:0001659,HP:0002360,HP:0003179,HP:0004970,HP:0005059,HP:0002705,HP:0012432,HP:0007800,HP:0001704"
```

Comment:
@glass-ships We should convert the multi-compare phenogrid example to message-sending mode rather than url-mode

Reply:
I think both should be supported and tested. This can have a button to trigger a message send like the search phenogrid example above it.

Reply:
Could we do a standalone testbed page just for multi-compare phenogrid via message?  That's the use case that we want to clearly provide to our IMPC friends.

Reply:
Creating a whole new route for that seems like overkill and a lot of duplication. I feel that the testbed code, and more importantly the readme code, clearly demonstrates how to use it.

However it looks like the phenogrid instructions were moved to `contributing.md` for some reason. I think there should be a `phenogrid.md` or something that clearly instructs how to use it via url and post message. And it could use more clarity, like strongly encouraging the message mode of operation for set to sets (to allow labels and such).

At most, I think we could have separate sections on the testbed for each mode of operation. Or just make message mode the default and have a button to test the url mode.

Reply:
Also somehow my additions/changes to the readme here were overwritten

https://github.com/monarch-initiative/monarch-app/pull/622/commits/9c666f8cc5fe90d2ae27762294f374dc20fc6717

Reply:
yeah we definitely missed a few things when we merged the documentation changes from main. 

I just re-added your changes to the section in what is now `contributing.md`

Reply:
A message mode example for multi-compare on testbed along with a phenogrid.md makes sense.

Reply:
I've updated the documentation and I think it's pretty clear how to use it.

In fact, we should be telling people to look at that documentation, and not our testbed. Our testbed should only be for our own testing purposes and not a place of record.

Reply:
If you really want an in-situ demo of the phenogrid widget to show people, a better idea would actually be to make a JSFiddle or something like that. That would also prove (and test) that it can work on a different domain (without CORS issues).